src/il_representations/envs/carla_envs.py

```diff
@@ -0,0 +1,121 @@
+import os
+import math
+import numpy as np
+import gym.spaces as spaces
+
+from il_representations.envs.config import (env_cfg_ingredient,
+                                            env_data_ingredient)
+from il_representations.envs.utils import stack_obs_oldest_first
+
+
+@env_data_ingredient.capture
+def _get_carla_data_opts(data_root, carla_demo_paths):
+    # workaround for Sacred issue #206
+    return data_root, carla_demo_paths
+
+
+@env_cfg_ingredient.capture
+def load_dataset_carla(task_name, carla_frame_stack, chans_first=True,
+                       n_traj=None):
+    data_root, carla_demo_paths = _get_carla_data_opts()
+
+    # load trajectories from disk
+    full_rollouts_path = os.path.join(data_root, carla_demo_paths[task_name])
+    trajectories = np.load(full_rollouts_path, allow_pickle=True)
```

Comment:
I take it that this is not the native D4RL data format? If so, how were these trajectories saved/generated?